runtime/src/runtime/actor_code.rs

```diff
@@ -2,24 +2,25 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use fvm_ipld_blockstore::Blockstore;
-use fvm_ipld_encoding::RawBytes;
+use fvm_ipld_encoding::ipld_block::IpldBlock;
 use fvm_shared::MethodNum;
 
 use crate::{ActorError, Runtime};
 
 /// Interface for invoking methods on an Actor
 pub trait ActorCode {
+    type Methods;
```

Comment:
Do we need this if `Methods` is not used in the trait?

Reply:
Is used by the macro dispatcher.